snuba/datasets/processors/search_issues_processor.py

```diff
@@ -99,6 +127,43 @@ def _process_user(
 
         return
 
+    def _process_tags(
+        self, event_data: IssueEventData, processed: MutableMapping[str, Any]
+    ) -> None:
+        tags_maybe = event_data.get("tags", None)
+        if not tags_maybe:
+            processed["tags.key"], processed["tags.value"] = [], []
+        tags: Mapping[str, Any] = _as_dict_safe(cast(Dict[str, Any], tags_maybe))
+        processed["tags.key"], processed["tags.value"] = extract_extra_tags(tags)
+        promoted_tags = {col: tags[col] for col in self.PROMOTED_TAGS if col in tags}
+        processed["release"] = promoted_tags.get(
+            "sentry:release",
+            event_data.get("release"),
+        )
+        processed["environment"] = promoted_tags.get(
+            "environment", event_data.get("environment")
+        )
+        processed["user"] = promoted_tags.get("sentry:user")
+        processed["dist"] = _unicodify(
+            promoted_tags.get("sentry:dist", event_data.get("dist")),
+        )
+
+    def _process_request_data(
+        self, event_data: IssueEventData, processed: MutableMapping[str, Any]
+    ) -> None:
+        request = event_data.get("request") or {}
+        http_data: MutableMapping[str, Any] = {}
+        extract_http(http_data, request)
+        processed["http_method"] = http_data["http_method"]
+        processed["http_referer"] = http_data["http_referer"]
+
+    def _process_sdk_data(
```

Comment:
Do you need this data?

Reply:
Not totally sure, but I think they're needed to support searching like this?
![Screen Shot 2023-01-05 at 5 04 57 PM](https://user-images.githubusercontent.com/101606877/210908933-c65a1356-3ac2-40d3-9e46-8c89db14b036.png)

Reply:
Yeah we do:
https://github.com/getsentry/snuba/blob/8fad778062c6e3db4be1bf36bc21ef08e61b7185/snuba/datasets/configuration/issues/entities/search_issues.yaml#L41-L42

Reply:
OK cool, just making sure.